modules/nf-core/vcfanno/tests/main.nf.test

```diff
@@ -0,0 +1,106 @@
+                input[2] = []
+                input[3] = [
+                    file("https://github.com/brentp/vcfanno/raw/master/example/exac.vcf.gz", checkIfExists: true),
+                    file("https://github.com/brentp/vcfanno/raw/master/example/exac.vcf.gz.tbi",checkIfExists: true)
+                ]              
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+
+    test("sarscov2 - [vcf, [], vcf], [], toml, [vcf, tbi]") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test_uncompressed', single_end:false ], // meta map
+                    file(params.test_data['sarscov2']['illumina']['test_vcf'], checkIfExists: true),
+                    [],
+                    file(params.test_data['sarscov2']['illumina']['test2_vcf'], checkIfExists:true)
+                ]
+                input[1] = file(params.test_data['homo_sapiens']['genome']['vcfanno_toml'], checkIfExists: true)
+                input[2] = []
+                input[3] = [
+                    file("https://github.com/brentp/vcfanno/raw/master/example/exac.vcf.gz", checkIfExists: true),
+                    file("https://github.com/brentp/vcfanno/raw/master/example/exac.vcf.gz.tbi",checkIfExists: true)
+                ]              
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+
+    test("sarscov2 - [vcf(gz), tbi, vcf], [], toml, [vcf, tbi] - stub") {
+
+        options "-stub"
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test_compressed', single_end:false ], // meta map
+                    file(params.test_data['sarscov2']['illumina']['test_vcf_gz'], checkIfExists: true),
+                    file(params.test_data['sarscov2']['illumina']['test_vcf_gz_tbi'], checkIfExists: true),
+                    file(params.test_data['sarscov2']['illumina']['test2_vcf'], checkIfExists:true)
+                ]
+                input[1] = file(params.test_data['homo_sapiens']['genome']['vcfanno_toml'], checkIfExists: true)
+                input[2] = []
+                input[3] = [
+                    file("https://github.com/brentp/vcfanno/raw/master/example/exac.vcf.gz", checkIfExists: true),
+                    file("https://github.com/brentp/vcfanno/raw/master/example/exac.vcf.gz.tbi",checkIfExists: true)
+                ]              
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    file(process.out.vcf[0][1]).name,
```

Comment:
```suggestion
                    file(process.out.vcf[0][1]).name,
```
why no md5sum here? it was in pytest

Reply:
That's the stub test (linting complains about empty files)

Reply:
Don't worry about it :wink: